db/1.10/modules/ROOT/pages/database-documentation.adoc

```diff
@@ -258,7 +258,7 @@ This operation allows delete operations to execute at various times using differ
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS |  A time unit that qualifies the `#queryTimeout` | `SECONDS` |
+** DAYS |  A time unit that qualifies the `#queryTimeout`. Values specified in nano, micro or milliseconds are rounded to seconds. | `SECONDS` |
```

Comment:
Global - Values specified in nanoseconds, microseconds, or milliseconds are rounded to seconds.